gradlew

```diff
@@ -163,11 +163,16 @@ fi
 
 CLASSPATH=$GRADLE_WRAPPER_JAR
 
-# Don't fork a daemon mode on initial run that generates local defaults.
-GRADLE_DAEMON_CTRL=
+# START OF SOLR CUSTOMIZATION
+# Generate gradle.properties if they don't exist
 if [ ! -e "$APP_HOME/gradle.properties" ]; then
-    GRADLE_DAEMON_CTRL=--no-daemon
+    "$JAVACMD" $JAVA_OPTS --source 11 "$APP_HOME/buildSrc/src/main/java/org/apache/lucene/gradle/GradlePropertiesGenerator.java" "$APP_HOME/gradle/template.gradle.properties" "$APP_HOME/gradle.properties"
```

Comment:
`buildSrc/src/main/java/org/apache/lucene/gradle/GradlePropertiesGenerator.java` I think this file wasn't checked in?

Reply:
The gitignore was ignoring everything lucene/. I've updated it to not ignore buildSrc\src\main\java\org\apache\lucene now